testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -801,13 +801,13 @@ def test_time_with_offset(self):
 
         assert_equal(ts.time, reftime + ref_offset)
 
-    def test_dt(self):
+    def test_dt(self, ref):
```

Comment:
looks like it doesn't know what ref is. Also you forgot a test_dt in the other test class. Did you run those tests on your local laptop?